taskcluster/ci/build/kind.yml

```diff
@@ -0,0 +1,64 @@
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - firefox_tv_taskgraph.build:transforms
+  - taskgraph.transforms.task:transforms
+
+jobs:
+  pr:
+    label: Firefox for Amazon's Fire TV - Build - Pull Request
```

Comment:
Nit: we can move this sentence as the description and let taskgraph generate the label for us. Here it would call it: `build-pr`. Same thing for the other tasks.

Can be done in a follow up.